### ISSUE TITLE
A question about ToT

### ISSUE THREAD

**Issue by kkramone** [MaartenGr/BERTopic#596]
First of all, thank you for this great model. I have a question about ToT. I have data in different texts for each day, for 365 days, from January 1st to December 31st. Let's say I set nr_bins = 12 for 12 months. Accordingly, as can be seen in the figure, do the words I see in the first month only belong to January?

Thanks
![Untitled](https://user-images.githubusercontent.com/27768674/177197553-369479a2-2599-43cc-a266-00e38d548a30.png)

**Comment by MaartenGr** [comment]
> Accordingly, as can be seen in the figure, do the words I see in the first month only belong to January?

Yes, the topics and their c-TF-IDF representations at timestamp *t* are re-calculated in order to get the words that only belong to that specific timestamp. However, this only applies if you have set both `global_tuning` and `evolution_tuning` to False. With `global_tuning`, the global representation gets averaged with the local representation, so any words can end up in that timestamp. With `evolution_tuning`, the representation at *t-1* gets averaged with *t*, so words will likely be a combination of those. 

You can find a bit more about dynamic topic modeling [here](https://maartengr.github.io/BERTopic/getting_started/topicsovertime/topicsovertime.html#example).

**Comment by kkramone** [comment]
Thank you my friend.

**Comment by MaartenGr** [comment]
No problem, glad I could help.
